### ISSUE TITLE
Is it possible to use Kvantum themes?

### ISSUE THREAD

**Issue by apandada1** [Zren/plasma-hud#24]
Is it possible for HUD to have blur like kvantum theme?
![1](https://user-images.githubusercontent.com/3063132/67160072-867e0700-f36a-11e9-8c13-cf847474bbeb.png)

However, the HUD has flat colors.

**Comment by sammyspark** [comment]
Agreed, this would be a very nice feature to have. If this is difficult to implement, even a rudimentary Konsole-style background translucency would be great, as so many people are using translucent terminals and/or menus already.

**Comment by apandada1** [comment]
I recently found [this repo](https://github.com/manilarome/rofi-spotlight) (for file search and web search) where `rofi` is somehow configured to use transparency and blur. It would be great if plasma-hud can use this feature.
![image](https://raw.githubusercontent.com/manilarome/rofi-spotlight/master/screenshots/rofi-web-ddg.png)

**Comment by Zren** [comment]
I don't think that's KWin. Rofi can probably draw a transparent background easy enough. The hard part is telling KWin to blur the contents below the window.

Would need to check these:
https://github.com/manilarome/rofi-spotlight/blob/master/rofi.rasi#L11-L23

**Comment by apandada1** [comment]
I installed rofi-spotlight in plasma. It is transparent, but does not use blur.
I also tried using the Force Blur KWin script (which is used [here](https://github.com/manilarome/blurredfox#im-using-kde-plasma-but-theres-no-blur-how-many-other-lies-have-i-been-told-by-the-council) to make firefox transparent) to force blur on "rofi" , but it did not work. However blur works with Firefox.

Is this because rofi does not use a traditional "window"?

**Comment by ren2r** [comment]
I don't know if it's this kind of thing that you want, but this kind of blur below it's possible to do it in kwin.
Although, I tested it sometime ago, I must have the code to enable it in one of my backups, the only thing needed was inject some rules in rofi window before populating it.
![Blur](https://user-images.githubusercontent.com/1781248/94501865-2b3ed080-01d9-11eb-9614-c0f9a2552d7c.png)
